src/custom/pages/About/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import useMarkdown from '../../hooks/useMarkdown'
+import { Title, Content, AppBodyMod } from './About.styled'
+import markdownContent from './about.md'
+import ReactMarkdown from 'react-markdown'
+
+export default function About() {
+  return (
+    <AppBodyMod>
+      <Title>About</Title>
+      <Content>
+        <ReactMarkdown>{useMarkdown(markdownContent)}</ReactMarkdown>
```

Comment:
You need to handle the error too, although if you load the MD using import, you won't need to handle the error. The compiler will take care of that

Reply:
Yeah, was thinking, a super simple approach would be to do a file import (no `fetch`). But as you mentioned, I could see use moving the Markdown files to a separate service.

For instance I saw that you could store Markdown files in Google Drive and then edit those with StackEdit.

Reply:
yep, thats a nice feature if we plan to have some dynamic content. For static content that don't change is overkill, and adds delays, but depends what we want to achieve